composer.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "cleantalk/typo3-cleantalk",
+  "type": "typo3-cms-extension",
+  "description": "Protect your site from spam",
+  "license": "GPL-2.0-or-later",
+  "version": "1.0.0",
+  "require": {
+    "typo3/cms-core": "^10.4 || ^11.0"
+  },
+  "require-dev": {
+    "cleantalk/antispam": "*",
+    "cleantalk/firewall": "*",
```

Comment:
`cleantalk/firewall` and `cleantalk/storage-handler` not necessary on this project. Remove these requires.